integration_tests/src/main/python/tpcds_test.py

```diff
@@ -35,5 +35,17 @@
 @allow_non_gpu(any=True)
 @pytest.mark.parametrize('query', queries)
 def test_tpcds(tpcds, query):
+  assert_gpu_and_cpu_are_equal_collect(
```

Comment:
Yes this change is some what unrelated and I can pull it out if people want me to.  I wanted to test all of the queries at a high scale factor to be sure that everything worked properly and I threw this change it with it because there were a lot of queries that were slow because of this.